frontend/src/scenes/session-recordings/templates/availableTemplates.ts

```diff
@@ -0,0 +1,147 @@
+                key: 'upgrade-subscribe-event',
+                description: 'The event that triggers the upgrade / subscribe flow.',
+            },
+        ],
+        categories: ['B2B'],
+    },
+    {
+        key: 'onboarding-flow',
+        name: 'Onboarding flow',
+        description: 'Watch how users onboard to your website. Look for any areas or steps that cause friction.',
+        variables: [
+            {
+                type: 'pageview',
+                name: 'Onboarding page URL',
+                key: 'onboarding-page-url',
+                description: 'Complete or partial URL',
+            },
+        ],
+        categories: ['B2B'],
+    },
+    {
+        key: 'feature-usage',
+        name: 'Feature usage',
+        description:
+            'Think of a feature you want to improve. Watch how users interact with it, and see where they get stuck.',
+        variables: [
+            {
+                type: 'event',
+                name: 'Feature event',
+                key: 'feature-event',
+                description: 'The event that indicates a user has interacted with the feature.',
+            },
+        ],
+        categories: ['B2B', 'B2C'],
+    },
+    {
+        key: 'purchase-flow',
+        name: 'Purchase flow',
+        description: 'Watch how users purchase from your website. Look for any areas or steps that cause friction.',
+        variables: [
+            {
+                type: 'pageview',
+                name: 'Purchase page URL',
+                key: 'purchase-page-url',
+                description: 'Complete or partial URL',
+            },
+        ],
+        categories: ['B2C'],
+    },
+    {
+        key: 'product-search',
+        name: 'Product search',
+        description:
+            'Watch how users search for products on your website. Look for any areas or steps that cause friction.',
+        variables: [
+            {
+                type: 'event',
+                name: 'Product search event',
+                key: 'product-search-event',
+                description: 'The event that indicates a user has searched for something on your website.',
+            },
+        ],
+        categories: ['B2C'],
+    },
+    {
+        key: 'experiment',
+        name: 'A/B test results',
+        description: 'Watch how users interact with your A/B test. Look for any areas or steps that cause friction.',
+        variables: [
+            {
+                type: 'flag',
+                name: 'Feature flag',
+                key: 'feature-flag',
+                description: 'The feature flag that you want to observe.',
+            },
+        ],
+        categories: ['More'],
+    },
+    {
+        key: 'scattershot',
+        name: 'Scattershot',
+        description: 'Watch all recent replays, and see where users are getting stuck.',
+        variables: [],
+        categories: ['More'],
+    },
+    {
+        key: 'person-property',
```

Comment:
the most common person property applied is `email` so we could prompt here to help folk understand